### PR TITLE
Fix ghosting of the eyes' pupils and nose

### PR DESCRIPTION
**Pull request by bubbliiiing**
1. The first facial fusion and the second facial fusion will easily cause ghosting of the eyes' pupils and nose.

before:
image

after:
image

2. I have changed 1.05 to 1.1 to prevent some images from not including the forehead

### CODE REVIEW

File: scripts/sdwebui.py [hunk 1]
```diff
@@ -412,11 +412,17 @@ def i2i_inpaint_call(
     p_img2img.script_args = init_default_script_args(p_img2img.scripts)
 
     for alwayson_scripts in modules.scripts.scripts_img2img.alwayson_scripts:
-        if alwayson_scripts.name is None:
-            continue
-        if alwayson_scripts.name=='controlnet':
-            p_img2img.script_args[alwayson_scripts.args_from:alwayson_scripts.args_from + len(controlnet_units)] = controlnet_units
-    
+        try:
+            if alwayson_scripts.name is None:
```
Comment (wuziheng) [inline scripts/sdwebui.py:416]: 这段代码是不是应该重构为

if alwayson_scripts.name is not None:
    if alwayson_scripts.name.lower() == 'controlnet':
        p_img2img.script_args[alwayson_scripts.args_from:alwayson_scripts.args_from + len(controlnet_units)] = controlnet_units

if alwayson_scripts.title() is not None:
    if alwayson_scripts.title().lower() == 'controlnet':
        p_img2img.script_args[alwayson_scripts.args_from:alwayson_scripts.args_from + len(controlnet_units)] = controlnet_units

而且这个try except过于滥用了

File: scripts/easyphoto_infer.py [hunk 2]
```diff
@@ -579,12 +580,13 @@ def easyphoto_infer_forward(
                     
                     # The edge shadows generated by fusion are filtered out by taking intersections of masks of faces before and after fusion.
                     # detect face area
-                    fusion_image_mask = np.int32(np.float32(face_skin(fusion_image, retinaface_detection, needs_index=[[1, 2, 3, 4, 5, 10, 11, 12, 13]])[0]) > 128)
-                    input_image_mask = np.int32(np.float32(face_skin(first_diffusion_output_image, retinaface_detection, needs_index=[[1, 2, 3, 4, 5, 10, 11, 12, 13]])[0]) > 128)
+                    fusion_image_mask   = np.int32(np.float32(face_skin(fusion_image, retinaface_detection, needs_index=[[1, 2, 3, 11, 12, 13]])[0]) > 128)
```
Comment (wuziheng) [inline scripts/easyphoto_infer.py:583]: 加个注释，这应该是(0,1)mask? # 0,1 mask of shape [c, h, w]? c==1?

**Comment by bubbliiiing** [comment]
api test pass
image

**Review by wuziheng — APPROVED** [review]
LGTM

**Pull request merged**